src/browser/index.tsx

```diff
@@ -44,4 +44,7 @@ function main(): void {
   );
 }
 
```

Comment:
You can declare this new property to prevent build errors. 
```suggestion
declare global {
  // eslint-disable-next-line @typescript-eslint/naming-convention
  interface Window {
    isElectron?: boolean;
  }
}

```

Reply:
Thanks for the suggestion